apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -217,14 +218,14 @@ func (a *Facade) UpdateApplicationsUnits(args params.UpdateApplicationUnitArgs)
 			continue
 		}
 
-		// always set to `false` to disable `jujuManagedUnits`
-		jujuManagedUnits := false
-		if jujuManagedUnits {
-			// TODO: remove here?
-			err = a.updateJujuManagedUnitsFromCloud(app, appUpdate.Units)
-		} else {
-			err = a.updateUnitsFromCloud(app, appUpdate.Units)
+		appConfig, err := app.ApplicationConfig()
+
+		// ingore `jujuManagedUnits`, log warning if it's still set to true
```

Comment:
Typo: ignore

(Also, comments should have start with capital letters (unless the first word is a variable name that is lowercase) and end with a full stop.)